aws/sdk/integration-tests/transcribestreaming/tests/test.rs

```diff
@@ -16,20 +15,25 @@ use aws_sdk_transcribestreaming::{Client, Config};
 use aws_smithy_client::dvr::{Event, ReplayingConnection};
 use aws_smithy_eventstream::frame::{DecodedFrame, HeaderValue, Message, MessageFrameDecoder};
 use bytes::BufMut;
-use futures_core::Stream;
 use std::collections::{BTreeMap, BTreeSet};
 use std::error::Error as StdError;
 
 const CHUNK_SIZE: usize = 8192;
 
 #[tokio::test]
 async fn test_success() {
-    let input_stream = stream! {
-        let pcm = pcm_data();
-        for chunk in pcm.chunks(CHUNK_SIZE) {
-            yield Ok(AudioStream::AudioEvent(AudioEvent::builder().audio_chunk(Blob::new(chunk)).build()));
-        }
-    };
+    let input_stream = FnStream::new(|tx| {
```

Comment:
FnStream is an implementation of generators–the network isn't involved in all use cases. By bounding it, you all the caller to avoid using more memory than is necessary